posts/wiki/Troubleshooting.md

```diff
@@ -9,15 +9,42 @@
 
 
 
-## Nvidia Video driver issues
-**Problem**: When running a machine with an Nvidia GPU, you notice graphical issues such as being stuck at a low resolution, failure to launch UI or slow rendering.
-</br></br>
+## Nvidia Video driver issues/poor performance
+**Problem**: When running a machine with an Nvidia GPU, you notice graphical issues such as being stuck at a low resolution, failure to launch UI, poor game performance or slow rendering.
 
-**Solution**: For any graphics card newer than the 10-series, run:
 </br></br>
+**Solution**: Install closed source Nvidia drivers per the following instructions: </br>
+**Note: Do not install drivers from Nvidia's website**
+</br></br>
+
+Depending on which Nvidia card you have, follow the instructions in A, B or C.
+
+***
+
+A) _If you have a 900 series card or newer_, run the following command:
+```bash
+sudo apt install nvidia-driver-latest
 ```
+***
+
+B) _If your card is older than the 900 series_, check [Nvidia's Website](https://www.nvidia.com/Download/index.aspx?lang=en-us) to see which driver corresponds to your graphics card.  If it is the latest Nvidia driver (525 at the time of writing), run the following command
+```bash
 sudo apt install nvidia-driver-latest
 ```
+
+Note: You can see what the latest Nvidia drivers packages are available in apt using the following command and looking for the highest numbered driver release
+```bash
+sudo apt search nvidia-driver
+```
+***
+C) _if you card is older than the 900 series and the driver listed on [Nvidia's Website](https://www.nvidia.com/Download/index.aspx?lang=en-us) is older than the latest driver_, do the following: </br> Install, through apt, the package that corresponds to your driver.  If the latest available driver is 470, run
+```bash
+sudo apt install nvidia-driver-470
```

Comment:
When installing a specific Nvidia driver, users need to install `disable-nouveau` as well.

Reply:
Addressed in next commit